server/src/main/java/fr/uga/l3miage/tp2/exo3/moduls/Commande.java

```diff
@@ -0,0 +1,24 @@
+package fr.uga.l3miage.tp2.exo3.moduls;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import java.util.Date;
+import java.util.Set;
+
+@Entity
+public class Commande {
+    @Id
+    private Long id;
+
+    private Date date;
+
+    private double montantTotal;
+
+    @ManyToOne
+    private Client client;
+
+    @OneToMany
+    private Set<Produit> produits;
```

Comment:
Très bien, vous avre du unidirectionnelles, cependant si vous écriver que ça vous avez une table d'association qui se créer. 

```suggestion
    @OneToMany
    @JoinColumn(name = "id_produit",referencedColumnName = "id")
    private Set<Produit> produits;
```

il vous faut le `@JoinColumn`